### PR TITLE
Update hubblestack_pulsar_win_config.yaml

### PR DESCRIPTION
**Pull request by mew1033**
Add some more Windows exclusions

### CODE REVIEW

File: hubblestack_pulsar/hubblestack_pulsar_win_config.yaml [hunk 1]
```diff
@@ -8,6 +8,19 @@ C:\Windows:
...
   exclude:
     - C:\Windows\System32
+    - C:\Windows\security\logs\diagnosis.log
+    - C:\Windows\ServiceProfiles\LocalService\AppData\Local\lastalive?.dat
+    - C:\Windows\WinSxS\FileMaps
+    - C:\Windows\security\logs
+    - C:\Windows\ServiceProfiles\LocalService\AppData\Local
+    - C:\Windows\security\templates\policies
+    - C:\Windows\CCM\Logs
+    - C:\Users\*\AppData\Local\VMware\vpx
```
Comment (anuragpaliwal80) [inline hubblestack_pulsar/hubblestack_pulsar_win_config.yaml]: I think it should be under Users section as:
C:\Users: {}
  exclude:
    - C:\Users\*\AppData\Local\VMware\vpx

File: hubblestack_pulsar/hubblestack_pulsar_win_config.yaml [hunk 2]
```diff
@@ -8,6 +8,19 @@ C:\Windows:
...
+    - C:\Windows\security\database
+    - C:\Users\*\AppData\Local\Mozilla\Firefox\Profiles
```
Comment (anuragpaliwal80) [inline hubblestack_pulsar/hubblestack_pulsar_win_config.yaml]: same as above.

File: hubblestack_pulsar/hubblestack_pulsar_win_config.yaml [hunk 3]
```diff
@@ -8,6 +8,19 @@ C:\Windows:
...
+    - C:\Users\*\AppData\Local\Temp
```
Comment (anuragpaliwal80) [inline hubblestack_pulsar/hubblestack_pulsar_win_config.yaml]: same as above.

File: hubblestack_pulsar/hubblestack_pulsar_win_config.yaml [hunk 4]
```diff
@@ -8,6 +8,19 @@ C:\Windows:
...
+    - C:\Windows\SoftwareDistribution
+    - C:\Windows\WinSxS
 C:\temp: {}
```
Comment (anuragpaliwal80) [inline hubblestack_pulsar/hubblestack_pulsar_win_config.yaml]: I think this was added for testing purpose and should be removed now.

**Comment by anuragpaliwal80** [comment]
Thanks for making these changes. 
General question: Are these folders C:/Users and C:/Windows are sufficient ?

File: hubblestack_pulsar/hubblestack_pulsar_win_config.yaml (hunk 4 above)
Comment (mew1033) [inline hubblestack_pulsar/hubblestack_pulsar_win_config.yaml]: Do you know who originally added it?

Comment (basepi) [inline hubblestack_pulsar/hubblestack_pulsar_win_config.yaml]: Pretty sure it was just on the list of "hey, here are some windows paths". I don't remember who created the original file. But I'm with @anuragpaliwal80, we probably don't need to watch temp

**Comment by mew1033** [comment]
Made the suggested changes

**Pull request merged**